railties/lib/rails/application.rb

```diff
@@ -400,6 +401,13 @@ def secrets
       end
     end
 
+    def credentials
+      @credentials ||= ActiveSupport::EncryptedConfiguration.new \
+        config_path: Rails.root.join("config/credentials.yml.enc"), 
```

Comment:
I'd like to see `bin/rails encrypted:edit FILE`, which will use the new `config/master.key` as its key, and which we can make accessible through something like `Rails.application.encrypted.config(path)[:key]`. Let's do that as a separate PR.